include/geom/face_quad4.h

```diff
@@ -159,6 +159,27 @@ class Quad4 : public Quad
 
 };
 
+class QuadShell4 : public Quad4
```

Comment:
> I think this new class should be in its own file.

I feel like that's overkill for a class with < 20 lines of code. It's overriding one function of the parent - I think that's related enough to justify keeping it in the same file and not having to monkey with the build system. Just my two cents.